src/app/components/app-layout/app-layout.component.html

```diff
@@ -37,15 +37,17 @@
     <div *ngIf="isAuthenticated" class="profile">
       <div class="btn-group" dropdown>
         <button class="btn dropdown-toggle" dropdownToggle id="button-basic" type="button"
-                aria-controls="dropdown-basic">
-          {{ name }} <img alt="" class="profile-picture" src="assets/janette-doe.png"> <span class="caret"></span>
+          aria-controls="dropdown-basic">
+          {{ name }} <img alt="" class="profile-picture" src="assets/janette-doe.png"> <span
+            class="caret"></span>
         </button>
-        <ul id="dropdown-basic" *dropdownMenu class="dropdown-menu"
-            role="menu" aria-labelledby="button-basic">
+        <ul id="dropdown-basic" *dropdownMenu class="dropdown-menu" role="menu"
+          aria-labelledby="button-basic">
           <li role="menuitem"><a class="dropdown-item" href="#">My profile</a></li>
           <li role="menuitem"><a class="dropdown-item" href="#">Account settings</a></li>
           <li class="divider dropdown-divider"></li>
-          <li role="menuitem"><a class="dropdown-item" (click)="signout()" routerLinkActive="active">Sign Out</a>
+          <li role="menuitem"><a class="dropdown-item" (click)="signout()" routerLinkActive="active">Sign
```

Comment:
Shouldn't click methods be prefixed with 'onClick'? I suggest we call this method onClickSignOut()

Reply:
Definitely, thanks. Method naming is not my strong point unfortunately.